src/atc_tools/helpers/ExtractEventhubBody.py

```diff
@@ -0,0 +1,62 @@
+import json
+import warnings
+
+from atc.spark import Spark
+from pyspark.sql.dataframe import DataFrame
+import pyspark.sql.types as t
+import pyspark.sql.functions as f
+
+
+class ExtractEventhubBody:
```

Comment:
This entire Tranformer has nothing to do with EvenHub, only with a binary encoded json column. Maybe change to name to something more general